### ISSUE TITLE
Implement URL encoding rules for DB url

### ISSUE THREAD

**Issue by vaibhavdobriyal** [retrography/OrientR#17]
#15 just involved handling for '@'.

Potentially full fledged URL encoding may be implemented.

Reference: http://www.blooberry.com/indexdot/html/topics/urlencoding.htm

**Comment by retrography** [comment]
Are you willing to have a stab at this, @vaibhavdobriyal?

**Comment by vaibhavdobriyal** [comment]
Yes I can give it a shot in near future; if we agree on the value this enhancement may deliver.
